.github/workflows/broken-linkcheck.yml

```
@@ -0,0 +1,30 @@
+---
+name: Broken linkcheck
```

Name sounds a little weird to me. Why not just

```suggestion
name: Linkcheck
```